cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -1577,7 +1604,9 @@ def _run_command_using_ssh(
                 response = []
                 error = [e]
 
-        return self._parse_command_results(status, response, error, secrets, **kwargs)
+        return self._parse_command_results(
+            status, response, error, secrets, ssh_job_pid=ssh_job_pid, **kwargs
```

Comment:
Actually I meant this function, you can pack ssh_job_pid into kwargs because it's not used in other commands, eg in the Python ones.